### DISCUSSION TITLE
Add option to limit label length in panels

### DISCUSSION THREAD

**Discussion by weeco** [grafana/grafana#38269] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

When I use table legends in Graph panels I'm often faced with very long labels. I'd like to be able to configure that labels longer than n characters shall be truncated.

E. g.

`super-hyper-crazy-long-label-with-useless-random-info-that-consumes-all-my-panel-width` => `super-hyper-crazy-lon...`

**Why is this needed**:

![image](https://user-images.githubusercontent.com/23424570/116757473-7094dc00-aa0e-11eb-8422-0f64da0292c5.png)

I'm aware that the legend width can be limited, however that causes the "avg"/ "current" etc to disappear because it's hidden behind a horizontal scrollbar / non visible area.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
